### DISCUSSION TITLE
Mineral scanner still has its function even the sonar is unpowered

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14551] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

you can detect where minerals are with an unpowered sonar

### Reproduction steps

1. take battery cell out of handheld sonar
2. turn on the sonar and mineral scanner
3. you can see mineral icons

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by pavel4win** [comment]
If we talk about realism.
A handheld sonar without a battery should show a blank screen and not work even as a passive sonar.
In passive scanning mode, a handheld sonar should discharge the battery, although 10 times slower than in active mode.
Unless we assume that it works from the heat of the human body, like a flashlight on the head of a diving suit.

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the unstable build.
